### PR TITLE
Submit Brightcom Bid Adapter

### PR DESCRIPTION
**Pull request by vyatsun**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder:  'brightcom',
  params: {
    publisherId: 2141020
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer: vladislavy@brightcom.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- [prebid/prebid.github.io#1186](https://github.com/prebid/prebid.github.io/pull/1186)

### CODE REVIEW

**Comment by vyatsun** [comment]
Hi @jsnellbaker

Could you please try again, and make sure you're sending the 'Origin' and 'User-Agent' headers?

If it still doesn't work for you, please share the request details.

File: modules/brightcomBidAdaper.js [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+  supportedMediaTypes: [BANNER],
+  isBidRequestValid,
+  buildRequests,
+  interpretResponse,
+  getUserSyncs
+};
+
+function buildRequests(bidReqs, bidderRequest) {
+  try {
+    const loc = utils.getTopWindowLocation();
```
Comment (jsnellbaker) [inline modules/brightcomBidAdaper.js]: This function is eventually going to be deprecated, so it shouldn't be used if possible.  

We have setup an alternative feature with this type of information that can be used instead.  The associated domain data can be accessed via the `bidderRequest.refererInfo`object.  

The following page provides some additional details on the object and its data points if you want to review further:  
http://prebid.org/dev-docs/bidder-adaptor.html#referrers

**Comment by vyatsun** [comment]
Hi @jsnellbaker

I've removed the utils.getTopWindowLocation() call.

**Comment by jsnellbaker** [comment]
Hi @vyatsun Thanks for making the updates; LGTM

**Pull request merged**